File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/org/openapitools/client/model/OrderStatisticsResponse.scala

package org.openapitools.client.model


case class OrderStatisticsResponse (
    /* total amount sold */
    _total: Long,
    /* total sold in last 24 hours */
    _last24h: Long,
    /* decimal average sales per second */
    _saleVelocityPerSeconds: Double
)
object OrderStatisticsResponse {
    def toStringBody(var_total: Object, var_last24h: Object, var_saleVelocityPerSeconds: Object) =
        s"""
        | {
        | "total":$var_total,"last24h":$var_last24h,"saleVelocityPerSeconds":$var_saleVelocityPerSeconds
        | }
        """.stripMargin
}

File: generated-sources/php-ze-ph/mojang-api/src/App/com/github/asyncmc/mojang/api/php/ze/ph/model/OrderStatisticsResponse.php
<?php

namespace com.github.asyncmc.mojang.api.php.ze.ph.server\com.github.asyncmc.mojang.api.php.ze.ph.model;

use Articus\DataTransfer\Annotation as DTA;

/**
 * A json object is returned with the total amount of copies sold, the amount of copies sold in the last 24h and how many sales there are per second.
 */
class OrderStatisticsResponse
{
    /**
     * total amount sold
     * @DTA\Data(field="total")
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $total;
    /**
     * total sold in last 24 hours
     * @DTA\Data(field="last24h")
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $last24h;
    /**
     * decimal average sales per second
     * @DTA\Data(field="saleVelocityPerSeconds")
     * @DTA\Validator(name="Type", options={"type":"float"})
     * @var float
     */
    public $sale_velocity_per_seconds;
}
